### PR TITLE
feat(registration): add parental option

### PR DESCRIPTION
**Pull request by ghost**
1. Implements the task: Display new consulting relation param "parental" in registration view [OB-5371](https://app.clickup.com/t/6656196/OB-5371)
2. Renames `ConsellingRelation` to `CounsellingRelation` (old mistype)
3. Removes unused `registrationDigi.ts` file (there’s an identical object in the `de.ts`)

This is how it’s supposed to look, but I just made it up in the DevTools.

image

### CODE REVIEW

File: src/enums/ConsellingRelation.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 export enum CounsellingRelation {
```
Comment (RunzelRosinchen) [inline src/enums/ConsellingRelation.ts]: Could you rename the file to "CounsellingRelation.ts"?
I know that typo existed before you made the changes but anyways 😅

**Review by RunzelRosinchen — CHANGES_REQUESTED** [review]

File: src/enums/ConsellingRelation.ts (hunk 1 above)
Comment (ghost) [reply src/enums/ConsellingRelation.ts]: Done!

**Comment by ghost** [comment]
@RunzelRosinchen @web-mi I have a feeling that the [src/resources/scripts/i18n/de/registrationDigi.ts](https://github.com/virtualidentityag/digiSucht-frontend/blob/develop/src/resources/scripts/i18n/de/registrationDigi.ts) is not used in this project anymore:

1. Registration uses `src/resources/i18n/de.ts` for localization
2. There’s identical `registrationDigi` object in the `de.ts`
3. If I remove the file, nothing happens, I don’t see it used anywhere

I’d like to remove it as a part of this PR, what do you think?

**Comment by RunzelRosinchen** [comment]
> @RunzelRosinchen @web-mi I have a feeling that the [src/resources/scripts/i18n/de/registrationDigi.ts](https://github.com/virtualidentityag/digiSucht-frontend/blob/develop/src/resources/scripts/i18n/de/registrationDigi.ts) is not used in this project anymore:
> 
> 1. Registration uses `src/resources/i18n/de.ts` for localization
> 2. There’s identical `registrationDigi` object in the `de.ts`
> 3. If I remove the file, nothing happens, I don’t see it used anywhere
> 
> I’d like to remove it as a part of this PR, what do you think?

If you're sure that the texts are identical then I would say yes, delete it. Otherwise we should check with Iva which texts are correct and adapt the de.ts before we delete the registrationDigi.ts.

**Comment by ghost** [comment]
> If you're sure that the texts are identical then I would say yes, delete it

Yes, they’re 100% identical. Thanks!

**Review by RunzelRosinchen — APPROVED** [review]

**Pull request merged**